crates/ruff_cli/src/args.rs

```diff
@@ -284,6 +284,7 @@ pub struct CheckArgs {
 pub enum HelpFormat {
     Text,
     Json,
+    Markdown,
```

Comment:
I think we should definitely choose a different name for this new format.

Because it's actually the `text` format that prints Markdown. The new format doesn't print Markdown but some rich text with escape sequences and special formatting. That it uses "mdcat" under the hood is only an implementation detail.

Reply:
Maybe `pretty`? (inspired by the git pretty formats)

Reply:
Opened #2797 to address this.

Reply:
👍🏻